Tests/AutoTests/FunctionalTest/functional-RadioNavigationalWarningsUpdateRNW.spec.ts

```diff
@@ -20,6 +20,7 @@ test.describe("Create new radio navigational warnings record", ()=> {
     await radioNavigationalWarnings.clearInput(radioNavigationalWarnings.reference);
     await radioNavigationalWarnings.clearInput(radioNavigationalWarnings.content);
     await radioNavigationalWarnings.editRNW();
+    await page.waitForTimeout(5000);
```

Comment:
Please don't use waitForTimeout().  Playwright docs state it should only be used for debugging (as per https://playwright.dev/docs/api/class-page#page-wait-for-timeout)

Reply:
Resolved